### PR TITLE
Add art to VapeShield readme

### PR DESCRIPTION
**Pull request by seano-vs**
**What this does:**

adds an image to the vapeshield repo.

_Side effects:_

We now have a binary in the repo, ew.

**Questions:**

Is this art satisfactory? WHY IS IT SO BIG??

**How to test:**

Go to this branch and look at the readme.

### CODE REVIEW

File: src/vape.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import schemathesis
 import os
 
-DEFAULT_TIMEOUT = 10
+DEFAULT_TIMEOUT = 1
```
Comment (zzolo) [inline src/vape.py:10]: This shouldn't be in here.

**Review by zzolo — CHANGES_REQUESTED** [review]
A couple things:

1) Resize the original image.  It is huge and there's no need to be.  Given that it's a portrait layout, maybe something that is like 400px wide.  You can also, use an HTML `` tag in Markdown and provide a width and/or height there.  Such as `Image of something or other`
2) It is probably a good idea to provide attribution for the image in the README.

File: src/vape.py (hunk 1 above)
Comment (seano-vs) [reply src/vape.py:10]: You're absolutely correct there.

**Comment by seano-vs** [comment]
> A couple things:

OK, @zzolo I made the changes. Thanks for the tip about img tags, had no idea about that.

**Review by zzolo — APPROVED** [review]

**Pull request merged**